packages/lib/src/index.ts

```diff
@@ -12,7 +12,7 @@ export type { HeatmapVisProps } from './vis/heatmap/HeatmapVis';
 export { default as Toolbar } from './toolbar/Toolbar';
 export { default as Separator } from './toolbar/Separator';
 export { default as Btn } from './toolbar/controls/Btn';
-export { default as DownloadBtn } from './toolbar/controls/DownloadBtn';
+export { default as LinkBtn } from './toolbar/controls/LinkBtn';
```

Comment:
I've made `DownloadBtn` more generic to just be a link button (i.e. with an `<a>` tag). `SnapshotBtn` now uses `LinkBtn`.